python/ola/Makefile.mk

```diff
@@ -58,16 +58,42 @@ python/ola/Version.py: python/ola/Makefile.mk configure.ac config/ola_version.m4
 # TESTS
 ##################################################
 
+python/ola/ClientWrapperTest.sh: python/ola/Makefile.mk
+	mkdir -p $(top_builddir)/python/ola
+	echo "export PYTHONPATH=${top_builddir}/python:${top_srcdir}/python; $(PYTHON) ${srcdir}/python/ola/ClientWrapperTest.py; exit \$$?" > $(top_builddir)/python/ola/ClientWrapperTest.sh
+	chmod +x $(top_builddir)/python/ola/ClientWrapperTest.sh
+
+python/ola/OlaClientTest.sh: python/ola/Makefile.mk
+	mkdir -p $(top_builddir)/python/ola
+	echo "export PYTHONPATH=${top_builddir}/python:${top_srcdir}/python; $(PYTHON) ${srcdir}/python/ola/OlaClientTest.py; exit \$$?" > $(top_builddir)/python/ola/OlaClientTest.sh
+	chmod +x $(top_builddir)/python/ola/OlaClientTest.sh
+
+python/ola/PidStoreTest.sh: python/ola/Makefile.mk
+	mkdir -p $(top_builddir)/python/ola
+	echo "export PYTHONPATH=${top_builddir}/python:${top_srcdir}/python; TESTDATADIR=$(srcdir)/common/rdm/testdata $(PYTHON) ${srcdir}/python/ola/PidStoreTest.py; exit \$$?" > $(top_builddir)/python/ola/PidStoreTest.sh
+	chmod +x $(top_builddir)/python/ola/PidStoreTest.sh
+
 dist_check_SCRIPTS += \
     python/ola/DUBDecoderTest.py \
+    python/ola/ClientWrapperTest.py \
     python/ola/MACAddressTest.py \
+    python/ola/OlaClientTest.py \
+    python/ola/PidStoreTest.py \
+    python/TestUtils.py \
```

Comment:
I think it would be good if we can leave this in python/ola and fix the other bits to avoid some other potential challenges.

Reply:
Won't adding this file to AC_CONFIG_LINKS avoid these issues?

Reply:
You're right, forgot about that hack.  Will revert that one.

Reply:
Done.